Makefile

```diff
@@ -114,7 +114,7 @@ clean-all: clean
 	make -C cpython clean
 	rm -fr cpython/build
 
-%.o: %.c $(CPYTHONLIB) $(wildcard src/**/*.h)
+%.o: %.c $(CPYTHONLIB) $(wildcard src/**/*.h src/**/*.js)
```

Comment:
@rth How do I make it so that `pyproxy.o` depends on `pyproxy.js` but not make it so the other `*.o` files depend on `pyproxy.js`?

Reply:
After this I think this PR is ready to merge.

Reply:
I'm not aware of a way to do that. Short of writing the compilation command for each *.o separately. Is it a significant issue though? Compilation of those is fast particularly with ccache..

Reply:
No it's not important, just a compulsive thing. In that case I guess I'll merge this.